pkg/auth/dependency/webapp/validate_provider_impl.go

```diff
@@ -23,6 +23,7 @@ var WebAppSchema = validation.NewMultipartSchema("").
 	Add(WebAppSchemaIDEnterLoginIDRequest, `
 		{
 			"type": "object",
+			"additionalProperties": false,
```

Comment:
Only enable `additionalProperties` in config. In web, I encountered error message 

```
map[kind:additionalProperties location:/gorilla.csrf.Token]
map[kind:additionalProperties location:/x_login_id_key]
```